### ISSUE TITLE
Live Link for Assignment

### ISSUE THREAD

**Issue by Husain010** [SusiragulR/rbac-workflow-assessment#1]
I would appreciate it if you could host this on Vercel and share a live link for the assignment!

-- Husain @ Complyance

**Comment by SusiragulR** [comment]
sure will host and share the link soon. Thanks

**Comment by Husain010** [comment]
Please ping once done.

**Comment by Husain010** [comment]
Any update on this? We will be closing review by today

**Comment by SusiragulR** [comment]
Will share the link within an hour and update you. Thanks

**Comment by SusiragulR** [comment]
https://rbac-workflow-assessment-e28o.vercel.app/

Hi Hussain,
I had attached the live link of the project above. I'm new to deployment in vercel and so it consumed long time. And I'm also facing an issue while fetching data from db in the live link but data gets uploaded to db as expected. If possible, kindly help me handling this. Hope to hear from you. 

Thanks and Regards,
Susiragul

**Comment by SusiragulR** [comment]
Sure Hussain. I haven't placed a signin tag at nav but I had placed it at the home page where the signIn alert links to sign in page in the alert message shown: "SignIn to continue".....and I had also placed an auto redirect option which redirects you to signIn page when you are in any other page than home page.

 And when I login and use the app the add Transaction feature works well and gets updated in db but when I click on the view transactions menu it doesn't fetch the transactions from db properly and shows up as no transactions found.

![Screenshot (51)](https://github.com/user-attachments/assets/a628e2c3-cda9-4186-8572-b4013f49f189)
![Screenshot (50)](https://github.com/user-attachments/assets/b6cf7d63-d8ea-4025-8b2f-7060b49fabfa)

**Comment by SusiragulR** [comment]
But when I run the app in local, it works properly and fetches the transactions and audit-log properly.

**Comment by Husain010** [comment]
Did you check the console? Try debugging. Also I cannot create an account, (which is what the signup page is what I was asking for) how do I signin then?

**Comment by SusiragulR** [comment]
Will check and update you soon. Thanks

**Comment by Husain010** [comment]
Oki

**Comment by SusiragulR** [comment]
Hi Hussain,
I had fixed all those errors and now the app is working completely fine and had checked all the endpoints.

https://susiragul-rbac-workflow-assessment.vercel.app/

Kindly use the above link and I had added a SignIn column too despite of automatic redirect to SignIn.

Use gitHub signIn for employee access. 

For, **manager and admin access**, **use manager and admin as username and password.**
